.circleci/config.yml

```diff
@@ -17,20 +17,6 @@ maybe-configure-production: &maybe-configure-production
           echo "export CF_APP=federalist-proxy" >> $BASH_ENV
           echo "export CF_VARS_FILE=./.cloudgov/vars/production.yml" >> $BASH_ENV
 
-maybe-configure-staging: &maybe-configure-staging
```

Comment:
`maybe-configure-staging` 😆

Reply:
The suspense was killing me so I had to drop it.